src/lock.rs

```diff
@@ -108,32 +123,38 @@ impl Lock {
         name: &str,
         version: &str,
         bump_func: Box<BumpFunc>,
-    ) -> Result<String, Error> {
```

Comment:
Since you can (and do) destructure the result with pretty name whenever you call the function, as far as I'm concerned it's fine not to make a type just for this. 
I think it's kind of a subjective preference, really! Couldn't blame anyone for making a type here.

Reply:
Good point. I'll leave it like that then